src/boot/ksdk1.1.0/devAS7262.c

```diff
@@ -204,9 +220,67 @@ readSensorRegisterAS7262(uint8_t deviceRegister)
 
 
 
+	/*
+	 *	This turns on the LED before reading the data
+	 */
+	returnValue = I2C_DRV_MasterSendDataBlocking(
+							0 /* I2C peripheral instance */,
+							&slave /* The pointer to the I2C device information structure */,
+							cmdBuf_LEDON /* The pointer to the commands to be transferred */,
+							2 /* The length in bytes of the commands to be transferred */,
+							NULL /* The pointer to the data to be transferred */,
+							0 /* The length in bytes of the data to be transferred */,
+							500 /* timeout in milliseconds */);
+
+	if (returnValue == kStatus_I2C_Success)
+	{
+		//...
```

Comment:
This regresses the version in master which checks the return value and returns an error. See the code you deleted.